tests/unit/dataSource/treeHierarchicalDataSource/tests.html

```diff
@@ -0,0 +1,662 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+	<title> Infragistics jQuery Data Source - Tests for general properties, API methods different than filtering, sorting or paging </title> 
```

Comment:
Please, fix the title. Instead of "Infragistics jQuery Data Source" it should be "Ignite UI Data Source".